app/helpers/pagination_helper.rb

```diff
@@ -1,14 +1,14 @@
 module PaginationHelper
   def next_page_link(url: nil, page: nil, total_pages: nil)
-    link_content = "Next <span class=\"visuallyhidden\">page</span> <span class=\"page-numbers\">#{page} of #{total_pages}</span>".html_safe
+    link_content = "#{I18n.t('support.pagination.next')} <span class=\"visuallyhidden\">page</span> <span class=\"page-numbers\">#{page} of #{total_pages}</span>".html_safe
```

Comment:
Looks like the words `page` and `of` are still hardcoded.  Maybe this could all be moved into one key?

```suggestion
    link_content = I18n.t("support.pagination_html", current_page: page, total_pages: total_pages)
```